src/components/PostPageDetail/elements/Content.js

```diff
@@ -227,7 +229,15 @@ const Content = ({
             </View>
           </View>
         ) : null}
-        {item && item.post_type === POST_TYPE_LINK && (
+        {item && item.post_type === POST_TYPE_LINK && images_url?.length > 0 ? (
+          <View style={styles.containerImage}>
+            <ImageLayouter
+              mode={FastImage.resizeMode.cover}
+              images={images_url || []}
+              onimageclick={onImageClickedByIndex}
+            />
+          </View>
+        ) : (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering of images based on the post type and the existence of images is a good improvement. It ensures that images are only displayed when they exist, preventing potential errors or empty image containers.
```diff
-        {item && item.post_type === POST_TYPE_LINK && (
-           <View
-             style={[
-               styles.newsCard,
+         {item && item.post_type === POST_TYPE_LINK && images_url?.length > 0 ? (
+           <View style={styles.containerImage}>
+             <ImageLayouter
+               mode={FastImage.resizeMode.cover}
+               images={images_url || []}
+               onimageclick={onImageClickedByIndex}
+             />
+           </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->